docs/source/examples.rst

```diff
@@ -1,35 +1,34 @@
 Examples
 ========
 
-Introduction
+Tidy3D Introduction
 ------------
 
 .. toctree::
     :maxdepth: 1
 
-    notebooks/StartHere
```

Comment:
Zongfu thinks it's a bit of a duplicate of the other intro notebooks and wants to streamline it. I guess it's a fair point. Or maybe we should name it better. Now it's a bit confusing since there is another start here tab above it. 
![image](https://github.com/flexcompute-readthedocs/tidy3d-docs/assets/116006359/99564d63-b2bb-4021-ba82-cc4f0e0f36d7)

Reply:
The purpose of this notebook is to be the simplest possible (meaningful) simulation one can make with Tidy3D. So maybe it would be good to re-label it something like that, for example instead of "start here" (maybe they shouldn't start there) it could be "Basic Simulation Template" or "Minimum Simulation"? If you want to remove too, that's ok, my feeling though is some people might just want something simple to look at to understand the basics and build off of.